src/pylyrics/plot_cloud.py

```diff
@@ -46,25 +49,37 @@ def plot_cloud(
 
     """
 
-    ## Exception Handeling to be added
+    # check input types
+    if type(song) != dict:
+        raise TypeError("song should be a variable of type dictionary.")
+    if not (type(file_path) == str and type(background_color) == str):
+        raise TypeError("Both file_path and background_color should be of type string.")
+    if not (type(max_font_size) == int and type(max_words) == int):
+        raise TypeError("Both max_font_size and max_words should be of type integer.")
+    if type(show) != bool:
+        raise TypeError("show only accepts True or False")
 
-    text = ""
-    # Create a string of all song lyrics
-    for artist, song_title in song.items():
-        raw_lyrics = extract_lyrics(artist, song_title)
-        clean_lyrics = clean_text(text=raw_lyrics)
-        text += " " + clean_lyrics  # Adding space for the end of lyrics
+    try:
+        text = ""
+        # Create a string of all song lyrics
+        for artist, song_title in song.items():
+            raw_lyrics = extract_lyrics(token, song, artist)
```

Comment:
same thing here. Since I do web scrapping, so the output might have changed please check.

Reply:
Adjust my code to include the changes.